src/services/bookstoreAPI.js

```diff
@@ -0,0 +1,33 @@
+export default class BookstoreAPI {
+  static BASEURL =
+    'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+
+  static ID = 'f2t7qderqVUd430ETjkz';
+
+  static getBooksFromAPI = async () => {
+    const url = `${this.BASEURL}/apps/${this.ID}/books`;
+    const response = url;
+    await fetch(url);
+    return response.json();
+  };
```

Comment:
- [ ] Your Books are not being stored to the API please try to connect it well to the API, I am able to add and remove but when I reload the page everything disappears so kindly fix that



https://user-images.githubusercontent.com/75126481/151667246-00d861dc-0156-4de7-a8c0-84628b7eeb35.MP4